www/include/configuration/configObject/host_categories/DB-Func.php

```diff
@@ -220,40 +221,37 @@ function multipleHostCategoriesInDB($hostCategories = [], $nbrDup = [])
                 if ($key2 != "hc_id") {
                     $fields[$key2] = $value2;
                 }
-                $val
-                    ? $val .= ($value2 != null ? (", '" . $value2 . "'") : ", NULL")
-                    : $val .= ($value2 != null ? ("'" . $value2 . "'") : "NULL");
             }
             $fields["hc_name"] = $hc_name;
             if (testHostCategorieExistence($hc_name)) {
-                $val
+                !empty($bindParams)
                     ? $query = "
                         INSERT INTO hostcategories
                         VALUES (NULL, :hc_name, :hc_alias, :level, :icon_id, :hc_comment, :hc_activate)"
                     : $query = null;
```

Comment:
PDO won't like a prepare(null)... on empty $bindParams, you should skip the next steps until line 244